modules/meeting/app/workers/meetings/pdf/export/agenda.rb

```diff
@@ -28,7 +28,7 @@
 # See COPYRIGHT and LICENSE files for more details.
 #++
 
-module Meetings::PDF
+module Meetings::PDF::Export
```

Comment:
I think the name "Export" to be somewhat weak. Both "Export" and "Minutes" are exports, so calling one subclass "Export" leads to confusion. 

E.g. "Default" would be better since that is also used in the UI.